src/iam-client-lib.ts

```diff
@@ -14,6 +14,7 @@
 //
 // @authors: Kim Honoridez
 
+import { IRoleDefinition, IAppDefinition, IOrganizationDefinition } from "@energyweb/iam-contracts";
```

Comment:
Firstly, my thinking is that it is best to define `IRoleDefinition`, IAppDefinition` and `IOrganizationDefinition` in the `iam-contracts`. In this way, they can be shared between `iam-client-lib` and `iam-cache-server`.
However, apps such as `Switchboard` may also want to use these interfaces. My thinking is that it is nice if Switchboard can just depend on `iam-client-lib` instead of needing to list `iam-contracts` in its package.json. So I'm therefore re-exporting in `iam-client-lib`. Does that make sense?

Reply:
Got it. Although this way Switchboard still depends on `iam-contracts`, because its code will need to change if these interfaces will be changed